frontend/src/app/index.js

```diff
@@ -1,48 +0,0 @@
-import 'babel-polyfill/browser';
```

Comment:
bootstrap.js has been renamed back to app.js since I switched back to having one big browserify bundle instead of multiple smaller ones. However the implementation is quite different.